.circleci/config.yml

```diff
@@ -92,17 +92,9 @@ commands:
                     key: cmake-<< parameters.cache-key >>-<< parameters.version >>
                     paths:
                         - << parameters.install-dir >>
-  uninstall-cuda:
-    description: "Uninstall CUDA-11.4"
-    steps:
-      - run:
-          name: "Uninstall CUDA-11.4"
-          command: |
-            sudo apt-get --purge remove "cuda*" "nsight*"
-            sudo apt-get --purge remove "*nvidia*"
 
   install-cudnn:
-    description: "Install CUDNN 8.8.0"
+    description: "Install CUDNN 8.8.1"
```

Comment:
Based on feedback and discussion with @HolyWu and @narendasan, the CuDNN was only upgraded in the patch version to avoid issues with PyTorch and CuDNN 8.9, while preserving the `libtorch.so` version installed by Torch. A subsequent upgrade to the newest CuDNN is to come when PyTorch support is more robust.